test/bundle.c

```diff
@@ -92,18 +92,23 @@ static void bundle_test_create_extract(BundleFixture *fixture,
 		gconstpointer user_data)
 {
 	gchar *outputdir;
+	RaucBundle *bundle;
 
 	outputdir = g_build_filename(fixture->tmpdir, "output", NULL);
 	g_assert_nonnull(outputdir);
 
-	g_assert_true(extract_bundle(fixture->bundlename, outputdir, TRUE, NULL));
+	g_assert_true(check_bundle(fixture->bundlename, &bundle, TRUE, NULL));
+	g_assert_nonnull(bundle);
+
+	g_assert_true(extract_bundle(bundle, outputdir, NULL));
 	g_assert_true(verify_manifest(outputdir, NULL, FALSE, NULL));
```

Comment:
Free the RaucBundle (in the other tests as well).

Reply:
Added missing `free_bundle()` calls